task5/list

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+read ext
+
+echo  | tee /c/work/test/task5/index.scv
+echo  | tee /c/work/test/task5/index.html
+
+files=$(find "$PWD" -type f -regex "^.*$ext")
+
+echo "<html>
+  <head>
+    <title>Bash list</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <h1>Result</h1>
+    <ul>" | tee -a /c/work/test/task5/index.html
+for f in $files
+  do 
+    filname=$(basename -- "$f")
+    extension="${filname##*.}"
+    filename="${filname%.*}"
+    echo "      <li>
+        <p>human.mp3<p>
```

Comment:
Here should be filename from $files i guess

Reply:
corrected